README.md

```diff
@@ -210,8 +210,7 @@ It is recommended to make all changes only in the  'platformio_override.ini', th
 
 ### using the pre-compiled .bin files
 
-The pre-compiled files can be found on the [github page](https://github.com/tbnobody/OpenDTU) in the tab "Actions" and the sub menu "OpenDTU Build". Just choose the latest build from the master branch (search for "master" in the blue font text but click on the white header text!). You need to be logged in with your github account to download the files.
-Use a ESP32 flash tool of your choice (see next chapter) and flash the `.bin` files to the right addresses:
+The pre-compiled binary files can be found here on the [github page behind "Releases"](./releases/:) (look at the right column). For a first installation on an ESP32, download `opendtu-generic.zip`, unpack and use a ESP32 flash tool of your choice (see next chapter) to flash the `.bin` files to the right addresses:
```

Comment:
This  relative link will not work. If a user is e.g. on a tag (e.g. https://github.com/tbnobody/OpenDTU/tree/v23.4.17) it will point to https://github.com/tbnobody/OpenDTU/tree/v23.4.17/releases which will give a 404

Reply:
I expected that to work (except for the colon at the end of the link, this is definitely a mistake...).
Okay, if in doubt we use a complete URL for the link.

Reply:
Hmm, but that could be avoided by referring to a static page:
`https://github.com/tbnobody/OpenDTU/releases`